logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/callback/ParticipantChangedHandlerImpl.kt

```diff
@@ -0,0 +1,55 @@
+package com.wire.kalium.logic.feature.call.callback
+
+import com.sun.jna.Pointer
+import com.wire.kalium.calling.callbacks.ParticipantChangedHandler
+import com.wire.kalium.logic.data.call.AvsClient
+import com.wire.kalium.logic.data.call.AvsClientList
+import com.wire.kalium.logic.data.call.Participant
+import com.wire.kalium.logic.data.id.QualifiedID
+import com.wire.kalium.logic.util.getDomain
+import com.wire.kalium.logic.util.removeDomain
+import org.json.JSONObject
+
+class ParticipantChangedHandlerImpl(
```

Comment:
What do you think of having this class renamed to `ParticipantChangedMapperImpl`?  But I see there is a callback handled. So another option maybe will be to create a Mapper class to help this `ParticipantChangedHandlerImpl` to transform the data, seen it seems to be the core of this logic.

Let me know if it makes sense to you =)

Reply:
Great idea, I thought on something like adding an inner class on `CallMapper` such as this:

```
val participantMapper = ParticipantMapper()

inner class ParticipantMapper() {
    fun doSomething() { .. }
}
```

and pass it down to `ParticipantChangedHandlerImpl`, such as:
```
val participantChangedHandler = ParticipantChangedHandlerImpl(
            participantMapper = callMapper.participantMapper,
            ...
) ...
```
?

Reply:
Then we could keep the mapping stuff inside `CallMapper` and on `ParticipantChangedHandlerImpl` we are really handling that it got changed but getting any mapped result back and passing back to the callback in it?

Reply:
👌🏻  Great proposal!

I have added `CallMapper.ParticipantMapper` and passed it down to this handler, so any mapping is done solely on the mapper itself and not here on the handler.

Thanks for the suggestion ❤️